scripts/combine/pullsAndImpacts.py

```diff
@@ -111,15 +125,15 @@ def plotImpacts(df, title, pulls=False, pullrange=[-5,5]):
         )
     return fig
 
-def readFitInfoFromFile(filename, group=False, sort=None, ascending=True, stat=0.0):
-    rf = uproot.open(filename)
-    name = "nuisance_group_impact_nois" if group else "nuisance_impact_nois"
+def readFitInfoFromFile(rf,filename, group=False, sort=None, ascending=True, stat=0.0, POI='Wmass', normalize=False):
+    # rf = uproot.open(filename)
+    # name = "nuisance_group_impact_nois" if group else "nuisance_impact_nois"
```

Comment:
Better to just delete than to comment out

Reply:
Thanks, done